### DISCUSSION TITLE
Thinking的代码块没有在窗口完整呈现，需要左右滑动才能看到（需要自动换行）

### DISCUSSION THREAD

**Discussion by zacmememe** [richards199999/Thinking-Claude#59] [Q&A]
![image](https://github.com/user-attachments/assets/277cb5ac-1498-41bf-bea7-29804a6a68d9)

如果想要思考代码块展现的文字在窗口自动换行，需要如何配置呢？感谢帮助!~

**Comment by lumpinif** [comment]
Hi, 请仔细查看首页README文档，如果想要增强效果您需要下载Chrome插件。
